process/process.go

```diff
@@ -706,6 +685,10 @@ func (p *Process) insertPropose(propose Propose) bool {
 		return false
 	}
 
+	if _, ok := p.TraceLogs[propose.Round]; !ok {
+		p.TraceLogs[propose.Round] = map[id.Signatory]bool{}
+	}
```

Comment:
I would suggest moving this to the moment where we actually modify `TraceLogs`, to keep relevant parts of code grouped together.
```suggestion
```